screens/SearchScreen.tsx

```diff
@@ -0,0 +1,186 @@
+import * as React from "react";
```

Comment:
isn't this a part of aarushi's PR?

Reply:
because I rebased on her branch. It would not conflict with the main.